core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -960,7 +960,7 @@ private CarbonCommonConstants() {
    * If set to GLOBAL_SORT, the sorting scope is bigger and one index tree per task will be
    * created, thus loading is slower but query is faster.
    */
-  public static final String LOAD_SORT_SCOPE_DEFAULT = "LOCAL_SORT";
+  public static final String LOAD_SORT_SCOPE_DEFAULT = "NO_SORT";
```

Comment:
why should change this default value?

Some more advices:
1. blank changes in this file is unnecessary. 
2. too many commits in this PR, better to merge them into if history does not matter. refer to command `git rebase` to merge the commits into one

Hope this PR can be merged soon

Reply:
resolved

Reply:
Actually thats the change coming from Master branch..when I rebased it. Please confirm..and I have squashed into one pull request.

Reply:
OK